packages/utils/tokens/package.json

```diff
@@ -32,13 +32,25 @@
   },
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
-    "@types/react-native": "^0.66.0",
     "@fluentui-react-native/scripts": "^0.1.1",
-    "react-native": "^0.66.0"
+    "@types/react-native": "^0.66.0",
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
   },
   "peerDependencies": {
-    "react-native": ">=0.66.0"
+    "react": "17.0.2",
+    "react-native": "^0.68.0"
   },
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "rnx-kit": {
```

Comment:
Curious if we need the rnx-kit field on every package? Not really sure what this does.

Reply:
This makes it so that when you do `npx rnx-dep-check --write`, it updates the dependencies on _every_ package with an rnx-config. Best case, this would mean that a RN version upgrade can be done by that single command.